_posts/2019-10-30-cockpit-206.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Cockpit 206
+author: Gundersanne
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-206
+category: release
+summary: Cockpit with machines, firewall, and login page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Machines: NIC deletion
```

Comment:
Let's please spell this out as "Network interface", or at least define an abbreviation.